java/java/JavaParser.g4

```diff
@@ -39,11 +39,12 @@ compilationUnit
     ;
 
 packageDeclaration
-    : annotation* PACKAGE qualifiedName ';'+
+    : annotation* PACKAGE qualifiedName ';'
     ;
 
 importDeclaration
-    : IMPORT STATIC? qualifiedName ('.' '*')? ';'+
+    : IMPORT STATIC? qualifiedName ('.' '*')? ';'
+    | ';'
```

Comment:
This is a better fix even though it makes the rule `compilationUnit` ambiguous (semicolons can be recognized as either `importDeclaration` or `importDeclaration`).

Reply:
Yes, and I have noticed this issue in the description of the [spec](https://docs.oracle.com/javase/specs/jls/se20/html/jls-19.html#jls-19-7) as well

Reply:
> 

The demo in the pr show the double semicolon after the statement of packageDeclaration and importDeclaration, but all the other statement with double semicolon ended is correct。So I suggest We should fix other situations。